windows/devices/uwb/include/windows/devices/uwb/UwbCxAdapter.hxx

```diff
@@ -11,7 +11,7 @@
 namespace windows::devices::uwb
 {
 /**
- * @brief TODO
+ * @brief Generate neutral type UwbSetAppConfigParameters from UwbSessionData TODO should this function be located in UwbCxAdapterDdiLrp.hxx?
```

Comment:
this file needs to be removed, not used anywhere

Reply:
Yes, I kept it so that we can use the code from it to implement a few functions for the new type. So this is intentional for now.